Sources/PackageGraph/PackageGraph.swift

```diff
@@ -33,8 +33,12 @@ public struct PackageGraph {
         return rootPackages.flatMap{$0.modules}.contains(module)
     }
 
+    /// The errors encountered while loading the package graph.
+    public let errors: [Swift.Error]
```

Comment:
Yep I agree, we need to unify all error, warning and notices. This is temporary until we get the  Diagnostic engine into SwiftPM.

Ah Got it @ reporting via delegate. I'll do that.